teams.md/src/components/include/migrations/botbuilder/user-authentication/python.incl.md

```diff
@@ -0,0 +1,237 @@
+<!-- auth-methods -->
+
+`sign_in` and `sign_out`
+
+<!-- example -->
+
+<Tabs groupId="sending-activities">
+  <TabItem value="Diff" default>
+    ```python
+    # highlight-error-start
+-   from botbuilder.core import (
```

Comment:
The diff view is supposed to help convey how similar concepts translate between the two sdks. They're less helpful if they're just set as two completely separate blocks. The first bit makes sense - remove these imports, add these other imports. How do you do that with the rest of it?